lib/components/StatusPercentage/StatusPercentage.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FC } from "react";
+
+import ProgressRing from "../Loader/ProgressRing";
+import { StatusPctWrapper } from "./StatusPercentage.styled";
+import { StatusPercentageProps } from "./StatusPercentage.types";
+
+export interface StatusPercentageSizeType {
+  radius: number;
+  stroke: number;
+}
+
+const StatusPercentage: FC<StatusPercentageProps> = ({
+  percentage,
+  size = "medium",
+  children,
+}) => {
+  const getRingSize = (): StatusPercentageSizeType => {
```

Comment:
this method is exposing a static data structure, why not have a data structure instead, like:
```
{
  small: { radius: 18, stroke: 4}
  medium: { radius: 18, stroke: 4}
}
``` 
?

Reply:
Good tip, thanks, I will change the solution ;-)